### DISCUSSION TITLE
AVD Session Host Module

### DISCUSSION THREAD

**Discussion by FallenHoot** [Azure/ResourceModules#3150] [Ideas]
It is rare that you edit the Hostpool after it has been created, but you do modify the AVD Session Host a lot with new images, VM SKU, and much more. Dependency can be Hostpool, but AVD Session should be decoupled for easier management. 

Currently, CARML has https://github.com/Azure/ResourceModules/tree/main/modules/Microsoft.DesktopVirtualization:
![image](https://user-images.githubusercontent.com/19664186/234584706-50b96251-8b4e-4d4f-b7e1-d233ec8bef3f.png)

Example here: https://github.com/Azure/avdaccelerator/blob/main/workload/bicep/modules/avdSessionHosts/deploy.bicep

**Comment by eriqua** [comment]
Hi @FallenHoot, thanks for opening this discussion and sorry for the late reply. 

CARML evolved to and has been rebranded as the Bicep version of Azure Verified Modules ([AVM](https://aka.ms/AVM)). 

All modules previously hosted by this repo have been updated to align with AVM specifications and published to the Public Bicep Registry. Their evolved source code can be found in the underlying repository ([BRM](https://aka.ms/BRM))

However, in order not to lose relevant discussions like this one, all CARML GitHub discussions are being converted to CARML GitHub issues, in order for the team to be able to further triage them and migrate them to AVM issues, if applicable.

Please refer to issue #4545 going forward.
